catalogs/catalog-hadoop/src/test/java/com/datastrato/gravitino/catalog/hadoop/TestHadoopCatalogOperations.java

```diff
@@ -533,6 +535,34 @@ public void testAlterFilesetProperties() throws IOException {
     }
   }
 
+  @Test
+  public void testFormalizePath() throws IOException {
+    String[] paths =
+        new String[] {
```

Comment:
Should we add a case for the relative path?

Reply:
Yeah, it can be, but hard to test because the working dir will be changed if we run the UT from different paths.

Reply:
Added.